src/common/utils/constants.jsx

```diff
@@ -0,0 +1,2 @@
+export const isProduction = process.env.REACT_APP_MODE === 'production';
+export const API_POINT = !isProduction ? process.env.REACT_APP_API_URL : 'none';
```

Comment:
위에서 말씀드린 것 과 같이 수정이 필요해보입니다 !

```suggestion
import dotenv from 'dotenv';

dotenv.config();

export const isProduction = process.env.REACT_APP_MODE === 'production';
export const API_POINT = !isProduction ? process.env.REACT_APP_API_URL : 'none';
```